lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+{
+  "guest": "Ospite",
+  "browse": "Esplora",
+  "search": "Cerca",
+  "library": "Libreria",
+  "lyrics": "Testi",
+  "settings": "Impostazioni",
+  "genre_categories_filter": "Filtra categorie o generi...",
+  "genre": "Genere",
+  "personalized": "Personalizzato",
+  "featured": "Featured",
+  "new_releases": "Nuove Release",
+  "songs": "Canzoni",
+  "playing_track": "In riproduzione {track}",
+  "queue_clear_alert": "Questo svuoterà la coda attuale. {track_length} brani saranno rimossi\nVuoi continuare?",
+  "load_more": "Carica altro",
+  "playlists": "Playlist",
+  "artists": "Artisti",
+  "albums": "Album",
+  "tracks": "Brani",
+  "downloads": "Download",
+  "filter_playlists": "Filtra le tue playlist...",
+  "liked_tracks": "Brani Preferiti",
+  "liked_tracks_description": "Tutti i tuoi brani preferiti",
+  "create_playlist": "Crea Playlist",
+  "create_a_playlist": "Crea una playlist",
+  "create": "Crea",
+  "cancel": "Annulla",
+  "playlist_name": "Nome Playlist",
+  "name_of_playlist": "Nome della playlist",
+  "description": "Descrizione",
+  "public": "Pubblico",
+  "collaborative": "Collaborative",
+  "search_local_tracks": "Cerca tra i brani locali...",
+  "play": "Riproduci",
+  "delete": "Elimina",
+  "none": "Nessuno",
+  "sort_a_z": "Ordina da A-Z",
+  "sort_z_a": "Ordina da Z-A",
```

Comment:
@OpenCode  does 'Ordina dalla A-Z' sound better?

Reply:
Yes. It's better.